internal/manifests/collector/config_replace_test.go

```diff
@@ -84,7 +84,7 @@ func TestPrometheusParser(t *testing.T) {
 		for _, scrapeConfig := range cfg.PromConfig.ScrapeConfigs {
 			assert.Len(t, scrapeConfig.ServiceDiscoveryConfigs, 1)
 			assert.Equal(t, scrapeConfig.ServiceDiscoveryConfigs[0].Name(), "http")
-			assert.Equal(t, scrapeConfig.ServiceDiscoveryConfigs[0].(*http.SDConfig).URL, "http://test-target-allocator:80/jobs/"+scrapeConfig.JobName+"/targets?collector_id=$POD_NAME")
+			assert.Equal(t, scrapeConfig.ServiceDiscoveryConfigs[0].(*http.SDConfig).URL, "http://target-allocator-service:80/jobs/"+scrapeConfig.JobName+"/targets")
```

Comment:
I could, but which ones would I substitute? I believe we should keep it explicitly written out to a degree since we want it to fail if the service name isn't what we expect.

Reply:
I used `naming.TAService()` for some. However, I believe some should be kept in raw format for testing purposes, right?